### PR TITLE
Add keras.backend.device() API for device scope

### PR DESCRIPTION
**Pull request by qlzh727**

### CODE REVIEW

File: keras/backend/jax/core.py
Comment (mattdangerw) [inline keras/backend/jax/core.py:321]: Should we expose something along the lines of `list_devices` as part of this work? Seems like it would be useful to avoid needing to write backend specific code there, but maybe already considered.

https://github.com/keras-team/keras/blob/9c675a9a45e5e8244163fea82efc6066722608a1/keras/backend/jax/distribution_lib.py#L14

Comment (fchollet) [reply keras/backend/jax/core.py:321]: +1

File: keras/backend/jax/core.py
Comment (fchollet) [inline keras/backend/jax/core.py:322]: Also, let's validate the string format so the error message is clear if users do something not supported like "cuda"

File: keras/backend/jax/core.py
Comment (qlzh727) [reply keras/backend/jax/core.py]: Done.

File: keras/backend/tensorflow/core.py
Comment (qlzh727) [reply keras/backend/tensorflow/core.py:285]: Yea, I think the cpu:0 should work for tensorflow, although the returned device will have CPU:0 in it. https://www.tensorflow.org/api_docs/python/tf/device

File: keras/backend/jax/core.py
Comment (qlzh727) [reply keras/backend/jax/core.py:319]: Ack, this will be a bit more work, will defer to a future PR.

**Comment by qlzh727** [comment]
Need to fix the test.

**Comment by qlzh727** [comment]
All the tests should pass now, the torch failure seems to be unrelated to my PR.

**Review by fchollet — APPROVED** [review]
LGTM, thank you!

**Comment by fchollet** [comment]
Will wait for the kokoro CI to clear before merging.

**Pull request merged**
